### ISSUE TITLE
Color contrast is insufficient - "Check Manually" button in dashboard

### ISSUE THREAD

**Issue by AmyJuneH** [backdrop/backdrop-issues#6118]
## Description of the bug
Color contrast is insufficient - the "Check Manually" button in the dashboard

The "Check Manually" button in the dashboard has insufficient color contrast.
"The color of the text and the color of the background are not in sufficient contrast to each other.
The contrast ratio should be at least 4.5:1 for normal text and 3:1 for large text.
Note: Contrasts are not relevant where content is hidden. But make sure that for hidden content, which becomes visible when receiving focus, the contrast ratio is sufficient.
Insufficient contrast between text and its background can give problems for users with low vision and color blindness, but it can also affect a lot of other users, leading to important text potentially being overlooked."

Results:
2.9 out of a required 4.5
Text color
 #ffffff / rgb(255,255,255)
Background color
 #97979e / rgb(151,151,158)

## Steps To Reproduce

To reproduce the behavior:
1. Go to '[/admin/dashboard](http://a11ydemo.simplo.site/admin/dashboard)'
3. Scroll down to the "AVAILABLE UPDATES". The Button is in that block.

**Comment by laryn** [comment]
This is in the Seven theme (which is the default admin theme currently) and the styling is used for secondary buttons more broadly throughout the interface:

```
a.button-secondary, input.button-secondary, button.button-secondary
```

<img width="593" alt="image" src="https://github.com/backdrop/backdrop-issues/assets/871421/50a920ae-a21c-45d9-8152-a924d33b6270">

and on hover:
<img width="593" alt="image" src="https://github.com/backdrop/backdrop-issues/assets/871421/eb482a32-b095-45af-9744-ce147f6f31d1">
